lib/prm/declaration/api.ex

```diff
@@ -54,4 +62,33 @@ defmodule PRM.DeclarationAPI do
     declaration
     |> cast(attrs, fields)
   end
+
+  defp declaration_search_changeset(%Declaration{} = declaration, attrs) do
+    fields = [
+      :patient_id,
+      :active,
+      :doctor_id,
+      :msp_id
+    ]
+
+    declaration
+    |> cast(attrs, fields)
+    |> validate_any_is_present(fields)
+  end
+
+  defp validate_any_is_present(changeset, fields) do
```

Comment:
Let's discuss this later on. I'm not familiar with `EView` facilities, and just would like this PR to be merged.

We can look into extracting common parts into `EView.Changeset.Validator*` also after doctor search is merged.